content/200-concepts/300-more/050-environment-variables/100-managing-env-file-manually.mdx

```diff
@@ -5,11 +5,85 @@ metaDescription: 'Learn how to manage .env files manually'
 tocDepth: 2
 ---
 
+<TopBlock>
+
+Although [Prisma creates an `.env` file for you upon installation](./#how-does-prisma-use-environment-variables), there can be times when you want to:
+
+- Bypass `.env` files and let Prisma use the system environment variables directly
+- Use `.env` files from a location that the Prisma CLI does not check [by default](./#using-env-files)
+- [Use multiple `.env` file](./using-multiple-env-files)
+
+</TopBlock>
+
+## Using the system environment directly
+
+Because Prisma reads from the systems environment when looking for environment variables, it's possible to skip using `.env` completely and create them manually on your local system. 
```

Comment:
```suggestion
Because Prisma reads from the system's environment when looking for environment variables, it's possible to skip using `.env` completely and create them manually on your local system. 
```